tests/testprovider/main.go

```diff
@@ -158,6 +158,11 @@ func (k *testproviderProvider) Parameterize(_ context.Context, req *rpc.Paramete
 // Invoke dynamically executes a built-in function in the provider.
 func (k *testproviderProvider) Invoke(_ context.Context, req *rpc.InvokeRequest) (*rpc.InvokeResponse, error) {
 	tok := req.GetTok()
+	if tok == "testprovider:index:returnArgs" {
```

Comment:
Is anything using this? Was the intention to write an integration test against this when the SDK support went in?